examples/custom-upgrade-behavior/main.tf

```diff
@@ -0,0 +1,8 @@
+provider "kubernetes" {}
+
+module "lacework_k8s_datacollector" {
+  source = "../../"
+
+  lacework_access_token      = "0123456789ABCDEF0123456789ABCDEF"
+  lacework_agent_autoupgrade = false
```

Comment:
Nice! For this example to be tested, we need to add it to the `scripts/ci_tests.sh` file inside the `TEST_CASES` array.

Reply:
Fixed!